Formula/p/pawk.rb

```diff
@@ -0,0 +1,25 @@
+class Pawk < Formula
+  desc "Python line processor (like AWK)"
+  homepage "https://github.com/alecthomas/pawk"
+  url "https://github.com/alecthomas/pawk/archive/refs/tags/v0.8.0.tar.gz"
+  sha256 "ede09939bfd51c680fecd0203d56fd0632aea3407cbf2ff23b50fb683f1a0cea"
+  license "MIT"
+
+  uses_from_macos "python"
+
+  def install
+    inreplace "pawk.py", /\A.*/, "#!/usr/bin/env python3"
```

Comment:
this can be done via `include Language::Python::Shebang`

Reply:
`rewrite_shebang detected_python_shebang(use_python_from_path: true), "pawk.py"`

Reply:
Thanks for the suggestion, I did not know such a method existed. I have made the required changes.